src/app/components/ColorSelector/ColorSelector.tsx

```diff
@@ -1,26 +1,49 @@
-import React from 'react';
-
-const colors = ['red', 'pink', 'grey', 'lime'];
+import React, { useState } from 'react';
+import styles from './ColorSelector.module.css';
+import { colors } from '../../config';
 
 type ColorSelectorProps = {
+  selectedColor: string;
   onSelectColor: (color: string) => void;
 };
 
-function ColorSelector({ onSelectColor }: ColorSelectorProps): JSX.Element {
+function ColorSelector({
+  selectedColor,
+  onSelectColor,
+}: ColorSelectorProps): JSX.Element {
+  const [isListOpen, setListOpen] = useState(false);
+
   return (
-    <label>
-      <span>choose color</span>
-      <select
-        name="colorSelector"
-        onChange={(event) => onSelectColor(event.target.value)}
+    <div className={styles.colorSelector}>
+      <h2>choose color</h2>
+      <div
```

Comment:
```suggestion
      <button
```
I would change it to a button to have full browser support. You can style the button like a div.